### PR TITLE
Add a speaker details page and link to it from the sessions

### PR DESCRIPTION
**Pull request by adrianbanks**
This pull request adds a speaker page, so that the details of an individual speaker can be shown/linked to. This page is linked to from each session.

To do this, I've extracted a partial view for showing the speaker details, and refactored some code in the controllers.

I've also styled the speakers page to be consistent with the sessions page so that a user's own profile will be highlighted and have a link to edit it.

### CODE REVIEW

**Comment by adrianbanks** [comment]
I've just noticed that this might address issue #34. Didn't mean it to, just thought it would be useful to link to the speaker.

File: DDDEastAnglia/Content/Site.css [hunk 1]
```diff
@@ -208,11 +208,11 @@ fieldset {
     display: none;
 }
 
-.session-actions {
-    text-align: right;    
+.actions {
```
Comment (alastairs) [inline DDDEastAnglia/Content/Site.css]: No need for this class, you can use `pull-right` from Bootstrap.

File: DDDEastAnglia/Views/Speaker/Details.cshtml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿@model DDDEastAnglia.Models.SpeakerDisplayModel
+
+@{
+    ViewBag.Title = "Details";
```
Comment (alastairs) [inline DDDEastAnglia/Views/Speaker/Details.cshtml:4]: Title should be set on the model really, not hard-coded in the view.

**Comment by alastairs** [comment]
Would benefit from some tests once Ian's change has gone in.

File: DDDEastAnglia/Views/Speaker/Details.cshtml (hunk 2 above)
Comment (adrianbanks) [inline DDDEastAnglia/Views/Speaker/Details.cshtml:4]: None of the other views currently do this, so I'd suggest that we target them all as a separate issue.

File: DDDEastAnglia/Content/Site.css (hunk 1 above)
Comment (adrianbanks) [inline DDDEastAnglia/Content/Site.css]: Done.

File: DDDEastAnglia/Views/Speaker/Details.cshtml (hunk 2 above)
Comment (alastairs) [inline DDDEastAnglia/Views/Speaker/Details.cshtml:4]: Ok.

**Pull request merged**